packages/pwa-kit-runtime/src/ssr/server/express.test.js

```diff
@@ -1242,25 +1233,46 @@ describe('SLAS private client proxy', () => {
     test('returns 403 if request is not for /shopper/auth endpoints', async () => {
         process.env.PWA_KIT_SLAS_CLIENT_SECRET = 'a secret'
 
-        const app = RemoteServerFactory._createApp(
-            opts({
-                mobify: {
-                    app: {
-                        commerceAPI: {
-                            parameters: {
-                                clientId: 'clientId',
-                                shortCode: 'shortCode'
-                            }
-                        }
-                    }
-                },
-                useSLASPrivateClient: true,
-                slasTarget: slasTarget
-            })
-        )
+        const app = RemoteServerFactory._createApp(opts(appConfig))
 
         return await request(app)
             .get('/mobify/slas/private/shopper/auth-admin/v1/other-path')
             .expect(403)
     }, 15000)
+
+    test('returns 403 if request is for /oauth2/trusted-system/* endpoint', async () => {
+        process.env.PWA_KIT_SLAS_CLIENT_SECRET = 'a secret'
```

Comment:
make sure to reset this variable at the end of each test

Reply:
This is done in line 1086 and 1119 of express.test.js